api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -500,8 +506,10 @@ public DbWorkspace getWorkspaceEnforceAccessLevelAndSetCdrVersion(
   }
 
   @Override
-  public List<UserRole> convertWorkspaceAclsToUserRoles(
-      Map<String, FirecloudWorkspaceAccessEntry> rolesMap) {
+  public List<UserRole> getFirecloudUserRoles(String workspaceNamespace, String firecloudName) {
+    Map<String, FirecloudWorkspaceAccessEntry> rolesMap =
+        getFirecloudWorkspaceAcls(workspaceNamespace, firecloudName);
+
```

Comment:
nit: UserRole could just define its own internal Comparator and implement Comparable so you don't need to do it below.